### PR TITLE
Fabric 1.19.4 base

### PR DESCRIPTION
**Pull request by Ascynx**

### CODE REVIEW

**Comment by iliabhz** [comment]
LGTM 👍

**Review by DuckySoLucky — APPROVED** [review]
lgtm

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "id": "skyblockcore",
+  "version": "${version}",
+
+  "name": "SkyBlock Core",
+  "description": "This is an example description! Tell everyone what your mod is about!",
+  "authors": [
+    "Skyblock Mod Developers"
+  ],
+  "contributors": [
+    "Ascynx"
```
Comment (My-Name-Is-Jeff) [inline src/main/resources/fabric.mod.json:12]: 💀

Comment (Ascynx) [reply src/main/resources/fabric.mod.json:12]: I copied it, enough to be put there

**Pull request merged**